### PR TITLE
Add blog for 'Clean-up Code with Duck-Typing'

### PR DESCRIPTION
**Pull request by sukima**

### CODE REVIEW

**Comment by treznick** [comment]
you've got a few typos in the blog post on tritarget. Also I think something like a more complex control flow (like a big `if...else if...else if...else if`) might be a better example for your duck typing example. Success and error handling seem a bit more distinct than a canonical shared interface might imply, such that it begs the question of whether or not that's the right abstraction.

Also "token" I think is the wrong word for what you're trying to get at. I'd better call it "abstraction", which is what you're doing when you write code.

Finally this talk is well worth a watch: https://www.youtube.com/watch?v=tJkoHFjoMuk

**Comment by sukima** [comment]
Just curious if there were different merge criteria between @maxx1128 pull requests and mine? His seem to get merged without delay.

Should the comments above by @treznick mean this is not a worth while topic or is covered elsewhere on youtube? If so I can close the PR.

**Comment by jnimety** [comment]
@sukima I think it's just that sometimes w all think someone else will merge. FWIW, hitting us up on Slack about a pending PR might get the ball rolling if things have stalled.

**Comment by maxx1128** [comment]
I thought it was more a matter of responding to feedback, since Devin also put an update on his on Slack. No one seemed to have any suggestions for changes or edits, so it got merged fast.

I assumed this was stalling since Tom recommended some edits, and there wasn't any updates on if they were being made or passed over or something.

Plus, it may have been faster since I've already written a lot of stuff for the site, so I'd had some form of "writer's tenure" or something haha.

**Comment by jnimety** [comment]
tl;dr it's not personal @sukima 😄

**Comment by treznick** [comment]
@sukima my comments were in no way meant as a blocker to merging. In writing, like in all things, there is always room for revision and change, and it's much easier to pick out blemishes than to see the quality of a piece overall. Glad it got merged :), and I don't really care about editorial process for this blog. We're a tech meetup, not the New Yorker. 

Like @jnimety said I think we all just assume someone else will merge. We've all got our day jobs and we don't often check this github repo to remember to keep the ball rolling. A helpful ping on slack is best :)

**Comment by maxx1128** [comment]
I think the rule we agreed on a while back was after 24 hours of the initial PR, if no one else merged and there's no one saying "this shouldn't be merged unless X Y or Z" then it's good to merge on your own. I'll add that to the site channel description.

**Comment by treznick** [comment]
Seems as good enough of a rule for me. Sometimes it's hard to get to something in 24 hours, but if so we can always edit the post/revert/etc. :)
